internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -89,6 +101,10 @@
     (OPTIONAL) Specify the URL to the Fluentd MSI package to install (default: "https://packages.treasuredata.com/4/windows/td-agent-4.1.0-x64.msi")
     .EXAMPLE
     .\install.ps1 -access_token "ACCESSTOKEN" -fluentd_msi_url https://my.host/td-agent-4.1.0-x64.msi
+.PARAMETER dotnet_tracing_msi_url
+    (OPTIONAL) Specify the URL to the SignalFx .NET Tracing MSI package to install (default: "https://github.com/signalfx/signalfx-dotnet-tracing/releases/download/v0.2.9/signalfx-dotnet-tracing-0.2.9-x64.msi")
```

Comment:
Don't know if there is a syntax for that, but, ideally the version should be a parameter or constant.

Reply:
Yeah I was kind of on the fence about that -- the reason that I didn't is because we're not doing that with fluentd in this script, and thought maybe it should be a separate effort for the entire script (possibly even handled by Jeff when he gets back). But not a big deal if we want to it now.

Reply:
It can also default to latest public release. Otherwise this script needs PR in every release.
https://github.com/signalfx/signalfx-dotnet-tracing#automated-download-and-installation-for-windows

Reply:
I'm aware of an effort that the Java folks worked on to automatically update the release version in the Collector with every release of the agent - https://signalfuse.atlassian.net/browse/APMI-3304, would be helpful to check in to see if a similar approach is possible for .NET as well.

Reply:
Yes, it's definitely possible in .NET also, although for me it's seems too minimal. The collector project should have too much knowledge about it's satellite projects. Instead of version only, you can download whole script.

Reply:
In principle, I like @RassK's idea: the .NET side controls the script that downloads and installs the .NET instrumentation the collector just need to know about the script. The point that seems a problem to me is regarding the Wizard: if we need a new option there or vice-versa, how we track this type of change?

Reply:
Since we are majorly targeting net new customers for this effort, the way we went about it for java was to expect the latest agent version being pulled down as part of the Collector deploy (latest version as well). As part of a future improvements, we can definitely provide an option to allow a manual intervention by customers to set the version as they would like from the wizard, reflecting on the installer instructions thereafter.

Reply:
I think that choosing the version is smaller issue than moving the script control back to the original repo.
The question here is not about the functionality but separation concerns between repos. The collector can still do what it's doing, but it first needs to download a install script which then downloads and installs a right version.
So the logic (control) how the instrumentation is installed, is maintained by the most knowledgeable part (in this specific case it would be the .net team). 
This also assures that you are downloading a right setup script for the right version. Handling this in the collector's setup script is not a good idea.

Reply:
@RassK I like this idea. There's a snippet in the markdown file that installs the latest version, but I'm not seeing a script in your repo. If you think it would be appropriate, would someone on your team be able to create it and put it in the right place?

Reply:
PR here https://github.com/signalfx/signalfx-dotnet-tracing/pull/635

Reply:
note: we may still need to standardize module naming if we adapt this pattern. Otherwise, there can be conflicts with other languages.

Reply:
[Splunk.SignalFx.DotNet.psm1](https://github.com/signalfx/signalfx-dotnet-tracing/releases/download/v0.2.9/Splunk.SignalFx.DotNet.psm1)

Module name is now updated to `Splunk.SignalFx.DotNet.psm1` for better compatibility.